external/llvm-project/mlir/lib/ExecutionEngine/CMakeLists.txt

```diff
@@ -197,23 +197,8 @@ if(MLIR_ENABLE_ROCM_RUNNER)
       set(ROCM_PATH $ENV{ROCM_PATH} CACHE PATH "Path to which ROCm has been installed")
     endif()
   endif()
-  set(HIP_PATH "${ROCM_PATH}/hip" CACHE PATH "Path to which HIP has been installed")
-  set(CMAKE_MODULE_PATH "${HIP_PATH}/cmake" ${CMAKE_MODULE_PATH})
-  find_package(HIP)
-  if (NOT HIP_FOUND)
-    message(SEND_ERROR "Building mlir with ROCm support requires a working ROCm and HIP install")
-  else()
-    message(STATUS "ROCm HIP version: ${HIP_VERSION}")
```

Comment:
You may still want to keep this message so that we can see the hip version.

Are you sure this is the find_package CONFIG mode? 
I did not find HIPConfig.cmake or HIP-config.cmake or something similar in the HIP project. But I did find FindHIP.cmake. I am really confused by the mode.
Or does ROCm provide hip configuration file?

Reply:
You're looking for `hip-config.cmake` lowercase. It matters
You'll find it in `/opt/rocm/lib/cmake/hip`

And I can't tell if we get the HIP version from that mode - or at least what variable it sets once we've found HIP

Reply:
Update: the variable I was looking for is `hip_VERSION`, and I'm adding a print for it internally, but not externally